charts/rundeck/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 description: Rundeck chart for Kubernetes
 name: rundeck
 home: https://github.com/rundeck/rundeck
-version: 1.4.0
+version: 1.4.1
```

Comment:
please make this 1.5.0 since this is not a patch / bugfix

Reply:
Sure, will do. Because it was non-breaking and would work with users' values file, I assumed it could qualify as a `patch` but `minor` makes sense.